akka-actor-typed-tests/src/test/java/jdocs/akka/typed/StashDocTest.java

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+package jdocs.akka.typed;
+
+//#import
+import akka.actor.typed.javadsl.StashBuffer;
+//#import
+
+import akka.Done;
+import akka.actor.typed.ActorRef;
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.testkit.typed.BehaviorTestkit;
+import akka.testkit.typed.TestInbox;
+import org.junit.Test;
+import org.scalatest.junit.JUnitSuite;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+public class StashDocTest extends JUnitSuite {
```

Comment:
we should either have all the doc specs with doc in the name or just rely on the package, atm we have a mixture (not introduced in this PR)

Reply:
I find it convenient to have a different name when opening it (search) in intellij